src/components/ChatApp/ChatApp.css

```diff
@@ -21,7 +21,7 @@ filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#183850', end
   overflow: hidden;
 }
 .app-bar{
-  background-image: url('../../images/susi-white.svg');
+  background-image: url('../../images/susi-logo-white.png');
```

Comment:
Please increase the size of the logo. The height is smaller compared to the present logo